util/ubuntu-setup

```diff
@@ -27,35 +27,16 @@ if cat /etc/os-release | grep -iq '^NAME=.*Debian'; then # dont add repositories
     SYSTEM="debian"
     ADD_REPOS=false
 elif cat /etc/os-release | grep -iq '^NAME=.*Ubuntu'; then # We are using a version of ubuntu...
-    if cat /etc/os-release | grep -iq '^VERSION=.*15.04'; then # Using Ubuntu 15.04
-        echo "Ubuntu 15.04 Detected..."
-        SYSTEM="ubuntu-15.04"
-        ADD_REPOS=false
-    elif cat /etc/os-release | grep -iq '^VERSION=.*15.10'; then # Using Ubuntu 15.10
-        echo "Ubuntu 15.10 Detected..."
-        SYSTEM="ubuntu-15.10"
-        ADD_REPOS=false
-    elif cat /etc/os-release | grep -iq '^VERSION=.*16.04'; then # Using Ubuntu 16.04
-        echo "Ubuntu 16.04 Detected..."
-        SYSTEM="ubuntu-16.04"
-        ADD_REPOS=false
-    elif cat /etc/os-release | grep -iq '^VERSION=.*16.10'; then # Using Ubuntu 16.10
-        echo "Ubuntu 16.10 Detected..."
-        SYSTEM="ubuntu-16.10"
-        ADD_REPOS=false
-    elif cat /etc/os-release | grep -iq '^VERSION=.*18.04'; then # Using Ubuntu 18.04
+    if cat /etc/os-release | grep -iq '^VERSION=.*18.04'; then # Using Ubuntu 18.04
         echo "Ubuntu 18.04 Detected..."
         SYSTEM="ubuntu-18.04"
         ADD_REPOS=false
-    elif cat /etc/os-release | grep -iq '^VERSION=.*19.04'; then # Using Ubuntu 19.04
-        echo "Ubuntu 19.04 Detected..."
-        SYSTEM="ubuntu-19.04"
-        ADD_REPOS=false
     elif cat /etc/os-release | grep -iq '^VERSION=.*20.04'; then # Using Ubuntu 20.04
         echo "Ubuntu 20.04 Detected..."
         SYSTEM="ubuntu-20.04"
         ADD_REPOS=false
     else
+        echo "Sorry, we only support 18.04 and 20.04 due to ROS2." >&2
```

Comment:
Foxy isn't backwards-compatible with 18.04, is it? And IIRC there were breaking API changes from Eloquent to Foxy; Foxy is the first time it's (supposed to) have a "stable" API. IMO we should only support Foxy on 20.04 here.

Reply:
Ok removed eloquent